frog/imports/ui/GraphEditor/SidePanel/fileConvert.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React, { Component } from 'react';
+import Form from 'react-jsonschema-form';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+import { Uploads } from '/imports/api/uploads';
+
+class fileConvert extends Component {
+  state: { url: string, copied: boolean, display: boolean };
+
+  constructor(props: {}) {
+    super(props);
+    this.state = { url: '', copied: false, display: false };
+  }
+
+  onChange = (e: { formData: { file: string } }) => {
```

Comment:
If I upload a big file, the UI still gets stuck until the file finishes to upload. I expected CollectionFS to work better for this, but I think you'd do better to use `Promise` as we had seen before.

Also, I'd love if when the file is chosen the url takes the value `Uploading. . .` and then the correct value when the upload is over. For this you can just `setState('Uploading. . .')` at the beginning of `onChange`